js/kronolith.js

```diff
@@ -3854,8 +3854,8 @@ KronolithCore = {
         }
         if (data.type == 'holiday') {
 			// checking if a holiday is already in the list
-            let check = this.getCalendarList(data.type).textContent.replace(/[^\w\s]/gi, '');
-            if (check === data.driver){
+            let check = this.getCalendarList(data.type).textContent;
+            if (check.includes(data.driver)){
```

Comment:
I think this will still fail if a calendar named `foobar` exists and want to create a new calendar named `foo` right?

Maybe I am wrong. I did not test it.

Reply:
yes true. Need to adapt for the fact that calendars have partly the same name, as in: Australia, AustraliaNWS and so on